ompi/mca/topo/base/base.h

```diff
@@ -194,6 +194,9 @@ OMPI_DECLSPEC int
 mca_topo_base_dist_graph_neighbors_count(ompi_communicator_t *comm,
                                          int *inneighbors, int *outneighbors, int *weighted);
 
+
```

Comment:
we already have a ```ompi_comm_neighbors_count``` subroutine that does the same think (at least at first glance)
that being said, your new subroutine only uses internal subroutine (vs ```MPI_*```) so i like it better.

should we only keep your subroutine ?